core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -77,6 +77,9 @@
       height: 56px;
       padding-top: 6px;
     }
+    .oppia-sidebar-menu-icon.fa {
+      font-size: 22px;
```

Comment:
Do you think this looks a little bit more balanced at 24px instead of 22px? Aside from that, this looks good to me. Thanks, @shaz13!